### PR TITLE
Fix cross compiling linux managed assemblies on windows

### PR DESCRIPTION
**Pull request by alexperovich**
This is needed to fix the source index.

### CODE REVIEW

**Comment by ViktorHofer** [comment]
@am11 can you please take a look at this PR?

File: eng/Configurations.props [hunk 1]
```diff
@@ -88,6 +88,7 @@
     <_runtimeOSFamily Condition="'$(_runtimeOSVersionIndex)' != '-1'">$(_runtimeOS.SubString(0, $(_runtimeOSVersionIndex)))</_runtimeOSFamily>
 
     <_portableOS>linux</_portableOS>
+    <_portableOS Condition="'$(_runtimeOS)' == 'win'">win</_portableOS>
```
Comment (am11) [inline eng/Configurations.props:91]: Please add this condition (`or '$(_runtimeOS)' == 'win'`) on line 94.

**Review by am11 — COMMENTED** [review]
LGTM, other than a nit about one case per OS.

**Review by safern — APPROVED** [review]
Other than @am11's comment.

**Comment by ViktorHofer** [comment]
@alexperovich I hope you don't mind that I updated your PR to make progress on this and getting closer to fixing source indexer.

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
GH shows the leg is still running even though AzDO reported the leg as being disconnected: https://dev.azure.com/dnceng/public/_build/results?buildId=920684&view=logs&jobId=cabfc1e4-b2ab-5fc9-5870-3b067d20e48f&j=cabfc1e4-b2ab-5fc9-5870-3b067d20e48f&t=c9eed12f-c879-5c97-d9e7-73b8dd159895.

cc @MattGal

**Pull request merged**